bin/fusor-update-repos

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import json
+import sys
+import getopt
+from subprocess import Popen, PIPE, STDOUT
+
+def usage(name):
+  print '\tUsage: %s -u <username> -p <password> -o <organiztion>\n' % name
+
+def main(argv):
+  found_u = False
+  found_p = False
+  found_o = False
+
+  try:
+    opts, args = getopt.getopt(argv,"hu:p:o:",["uname=","passwd=","org"])
```

Comment:
FYI, in future you could look at using argparse, it handles required parameters

https://docs.python.org/2/library/argparse.html#required